app/components/minister-status.js

```diff
@@ -0,0 +1,22 @@
+import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
+
+export default class MinisterStatusComponent extends Component {
+  get isActiveMinister() {
+    let endDate = this.args.endDate;
+
+    if (!endDate) {
+      // No end date set, so the mandatory is still active
+      return true;
+    } else {
+      return isDateInTheFuture(endDate);
+    }
+  }
+}
+
+function isDateInTheFuture(date) {
```

Comment:
I think we use this elsewhere already? Maybe we can export this from a `app/utils/date.js` file or something?

Reply:
Yes, that's why I'm suggesting creating a generic component and pass only the end date.

Reply:
Ok! Didn't know where we used the other one. Yea in that case renaming it to PositionStatus makes sense!